/ros2_standard_robot_pp:
  ros__parameters:
    device_name: /dev/ttyACM0
    baud_rate: 115200
    flow_control: none
    parity: none
    stop_bits: "1"
    send_sof: 0x5A
    receive_sof: 0x5A
